### ISSUE TITLE
Backend type gcs is not supported referencing remote tfstate

### ISSUE THREAD

**Issue by andrewnazarov** [variantdev/vals#38]
After https://github.com/variantdev/vals/issues/34 was improved I've finally had a chance to test remote backend when referencing tfstate again, and now I see the real error instead of panic:

```
err: expand tfstate:////builds/devops/k8s/shared-services/helmfile-shared-services-staging/.terraform/terraform.tfstate/output.external-ip.value: reading tfstate for /builds/devops/k8s/shared-services/helmfile-shared-services-staging/.terraform/terraform.tfstate/output.external-ip.value: backend type gcs is not supported
changing working directory back to "/builds/devops/k8s/shared-services/helmfile-shared-services-staging"
in ./helmfile.yaml: in .helmfiles[0]: in /builds/devops/k8s/shared-services/helmfile-shared-services-staging/.helmfile/cache/https_my_domain_com_my_project_k8s_shared-services_helmfile-common_git.ref=v0.37/helmfile.yaml: expand tfstate:////builds/devops/k8s/shared-services/helmfile-shared-services-staging/.terraform/terraform.tfstate/output.external-ip.value: reading tfstate for /builds/devops/k8s/shared-services/helmfile-shared-services-staging/.terraform/terraform.tfstate/output.external-ip.value: backend type gcs is not supported
```

In my helmfile I'm setting this the following way:

```
...
service:
  loadBalancerIP: {{- if ne .Environment.Name "default" }} ref+tfstate:///{{ env "PATH_TO_TF_STATE" }}/output.external-ip.value {{- end }}
...
```

and in CI I'm just doing:

```
...
export PATH_TO_TF_STATE=$(pwd)/.terraform/terraform.tfstate
...
```

Haven't checked on a plain vals though, I'm using it as a part of helmfile
```
HELMFILE_VERSION=v0.130.0
```

**Comment by mumoshu** [comment]
@andrewnazarov Thanks for the update! Turns out the package "tfstate-lookup" which vals uses for querying tfstate doesn't support the GCS backend yet:

https://github.com/fujiwara/tfstate-lookup/blob/6da1fba26ab4944e2df09ff549a8f6f9812abae8/tfstate/remote.go#L18

Looks like we'd better contribute it 😃

**Comment by andrewnazarov** [comment]
As a first step, I've opened an issue:)
https://github.com/fujiwara/tfstate-lookup/issues/13

**Comment by dex4er** [comment]
Hi. I made fujiwara/tfstate-lookup#21 and I hope it will be accepted by project owner.

**Comment by mumoshu** [comment]
@andrewnazarov @dex4er Can this be safely closed since #47 and #49 have been merged?

**Comment by andrewnazarov** [comment]
Yes, sure. Thanks for the contribution!
